apis/projectcontour/v1/httpproxy.go

```diff
@@ -171,6 +171,26 @@ type AuthorizationServer struct {
 	//
 	// +optional
 	FailOpen bool `json:"failOpen,omitempty"`
+
+	// BufferSettings specifies configuration for buffering request data sent to AuthorizationServer
+	// +optional
+	BufferSettings *AuthorizationServerBufferSettings `json:"bufferSettings,omitempty"`
```

Comment:
interested in what others think, but I'd prefer a field name more like `IncludeRequestBody` which I think is more obvious to users, e.g.:

```yaml
authorizationServer:
  includeRequestBody:
    maxRequestBytes: 1024
    allowPartialMessage: true
...
```

Reply:
I think that's much clearer in YAML form, great idea.

Reply:
yes.. renaming makes sense. 
Done